src/algorithms/tracking/TrackSeeding.cc

```diff
@@ -276,23 +271,34 @@ std::pair<float, float> eicrecon::TrackSeeding::findPCA(std::tuple<float,float,f
   return std::make_pair(xmin,ymin);
 }
 
-int eicrecon::TrackSeeding::determineCharge(std::vector<std::pair<float,float>>& positions) const
+int eicrecon::TrackSeeding::determineCharge(std::vector<std::pair<float,float>>& positions, const std::pair<float,float>& PCA, std::tuple<float,float,float>& RX0Y0) const
```

Comment:
Ideally, this function should accept vectors from `Acts::`. To not drag this on, let's agree that someone (you?) would kindly do such a refactor in a follow up PR.